source/RoslynAnalyzers/MessageContractAnalyzers.cs

```diff
@@ -107,18 +106,12 @@ static bool CheckMessageTypeProperties(SyntaxNodeAnalysisContext context, TypeDe
                 // only validate public properties
                 if (!propDec.Modifiers.Any(m => m.IsKind(SyntaxKind.PublicKeyword))) continue;
 
-                // we have a number of places where we treat collection types differently; front-load that info
-                var isCollectionType = IsCollectionType(context, propDec);
```

Comment:
I _think_ we can also drop the `IsCollectionType` function now?

Reply:
Yeah, it's not used anymore. I chose to keep it around because it seems like something we might need again in the near future, and it serves as quite a useful reference when trying to do other type-checking things.